api/v1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
qHmm... looking at this, I wonder whether we should also have a job guarding that these autogenerated files stay up to date? Shall we bump our generators sometimes? `controller-tools@v0.3.0` is ancient...